packages/grafana-ui/src/components/DataLinks/DataLinksContextMenu.tsx

```diff
@@ -24,11 +27,27 @@ export const DataLinksContextMenu: React.FC<DataLinksContextMenuProps> = ({ chil
     cursor: context-menu;
   `;
 
-  return (
-    <WithContextMenu getContextMenuItems={getDataLinksContextMenuItems}>
-      {({ openMenu }) => {
-        return children({ openMenu, targetClassName });
-      }}
-    </WithContextMenu>
-  );
+  if (linksCounter > 1) {
+    return (
+      <WithContextMenu getContextMenuItems={getDataLinksContextMenuItems}>
+        {({ openMenu }) => {
+          return children({ openMenu, targetClassName });
+        }}
+      </WithContextMenu>
+    );
+  } else {
+    const linkModel = links()[0];
+    return (
+      <a
+        href={linkModel.href}
+        onClick={linkModel.onClick}
+        target={linkModel.target}
+        title={linkModel.title}
+        style={{ display: 'flex' }}
+        aria-label={selectors.components.DataLinksContextMenu.singleLink}
+      >
```

Comment:
think rendering anchor will make this tricky to use in the PieChart where we it's inside an svg

Reply:
but not sure what the alternative is

Reply:
@dprokop any ideas for how we can make this work when inside an svg?